examples/simulation/Tutorial1/macros/run_tutorial1.C

```diff
@@ -160,4 +150,27 @@ void run_tutorial1(Int_t nEvents = 10,
     cout << "Macro finished successfully." << endl;
 
     // ------------------------------------------------------------------------
+
+    auto file = TFile::Open(outFile.data());
+    auto tree = static_cast<TTree*>(file->Get("cbmsim"));
+    auto nofE = tree->GetEntries();
+    auto nofT = tree->Draw("MCTrack.fPx", "", "goff");
+    auto nofP = tree->Draw("TutorialDetPoint.fX", "", "goff");
+    cout << endl << "Output tree has " << nofE << " events, " << nofT << " tracks, " << nofP << " points." << endl;
+    if (nofE < nEvents) {
+        cout << "Not enough events (" << nofE << " < " << nEvents << ") in the output tree." << endl;
+        return 1;
+    }
+    if (nofT < nEvents * nofPart) {
+        cout << "Not enough tracks (" << nofT << " < " << nEvents * nofPart << ") in the output tree." << endl;
+        return 1;
+    }
+    if (nofP < nEvents * nofPart) {
+        cout << "Not enough points (" << nofP << " < " << nEvents * nofPart << ") in the output tree." << endl;
+        return 1;
+    }
+    cout << "Simulation successful." << endl;
+    file->Close();
```

Comment:
```suggestion
```

This is probably not needed, because the unique_ptr will destruct (and thus close) the file, hopefully.